pkg/nanny/nanny.go

```diff
@@ -104,3 +104,14 @@ func (n *Nanny) GetTimer(name string) *Timer {
 func (n *Nanny) SetTimer(name string, timer *Timer) {
 	n.timers.Set(name, timer)
 }
+
+// GetTimers returns a slice of currently open timers
+func (n *Nanny) GetTimers() []*Timer {
+	timers := make([]*Timer, n.timers.Len())
```

Comment:
Better is to create it with capacity of `len(items)`, and than juse append. It has the same effect on reducing allocations but is more readable.

Reply:
And also, here we don't expect to have many timers so the allocations here would be negligible. Optimize after testing and actually measuring 😉

Reply:
When the `timers` slice is initialized with length n and then just `timers = append(timers, timer)`, the `timers` slice ends up with a length of n+1 starting with n empty items.